src/main/resources/webapp/js/directives.js

```diff
@@ -847,6 +843,7 @@
                                 scope.form.dto.participantName = rowData.participantName;
                                 scope.form.dto.visitType = rowData.visitType;
                                 scope.form.dto.plannedDate = rowData.plannedDate;
+                                scope.form.dto.actualDate = rowData.actualDate;
                                 scope.form.dto.startTime = rowData.startTime;
                                 scope.form.dto.visitId = extraRowData.visitId;
                                 scope.form.dto.ignoreDateLimitation = extraRowData.ignoreDateLimitation;
```

Comment:
you should calculate min and max dates for actual date like it is done for planned date few line below